schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/SchemaRegistryResource.java

```diff
@@ -983,4 +1019,79 @@ public Response deleteSchemaVersion(@ApiParam(value = "Schema name", required =
             return WSUtils.respond(Response.Status.INTERNAL_SERVER_ERROR, CatalogResponse.ResponseMessage.EXCEPTION, ex.getMessage());
         }
     }
+
+    @GET
+    @Path("/branches")
```

Comment:
Is not that good to hive branches for each schema metadata instead of returning all branches as user may not be able to get which schema metadata that branch belongs to?

Reply:
I will take this as a github issue along with the API for generating aggregated schema